### ISSUE TITLE
Add support to define resources inside directories

### ISSUE THREAD

**Issue by chihaiaalex** [F5Networks/terraform-provider-bigip#411]
### Is your feature request related to a problem? Please describe.

It would be useful for the provider to support the definition of resources inside directories.

Example:

```
resource "bigip_ltm_pool" "pool" {
  name = "/${var.partition}/${var.directory}/${var.pool_name}"
  load_balancing_mode = "round-robin"
  monitors = [
    "/Common/tcp"
  ]
  allow_snat = "yes"
  allow_nat = "yes"
}
```

Right now the provider expects that all resources are named: ``/partition/resource-name``

### Additional context
Internally, we have a multi datacenter HA/DR setup and we are using a ``sync`` directory for the resources that we want to be synced cross data center.

### Describe the solution you'd like

I would like to be able to use directories in resource names, the same way we are currently doing it in BigIP manually.

### Describe alternatives you've considered

For my particular usecase, the alternative would be to apply the same configuration twice, with different provider configurations, for both BigIP instances.

**Comment by nmenant** [comment]
Tracking this internally with INFRAANO-168.

**Comment by papineni87** [comment]
@chihaiaalex  Added support to define resources inside directory for below resources in release v1.7

pool
virtual server
pool attachment

Let me know if that works

**Comment by RavinderReddyF5** [comment]
closing this issues as it is fixed in v1.7.0
